docs/materials/htc/part1-ex3-jobs.md

```diff
@@ -48,15 +48,15 @@ The lines of the submit file have the following meanings:
 | `output`     | The filename where HTCondor will write the standard output from your job.                                                                                                  |
 | `error`      | The filename where HTCondor will write the standard error from your job. This particular job is not likely to have any, but it is best to include this line for every job. |
 | `log`        | The filename where HTCondor will write information about your job run. Technically not required, it is a **really** good idea to have a log file for every job.            |
-| `request_*`  | Tells HTCondor how many `cpus` and how much `memory` and `disk` we want, which is not much, because the 'hostname' executable is pretty simple                             |
+| `request_*`  | Tells HTCondor how many `cpus` and how much `memory` and `disk` we want, which is not much, because the 'hostname' executable is pretty small                             |
```

Comment:
This is fine, but I would change the word “pretty” here to nothing or “very” or “relatively”.  Try defining the word “pretty” as used in this context to a non-native speaker…